### ISSUE TITLE
Feature Request: Add edit button

### ISSUE THREAD

**Issue by CorellanStoma** [GooseMod/OpenAsar#29]
Add a button that opens the settings.json with the default program.

**Something like this:**

![image](https://user-images.githubusercontent.com/58918358/158085626-0fd6fc65-4dd1-4687-a50d-049717a40814.png)

**Comment by KraXen72** [comment]
+1

**Comment by Assassinduck** [comment]
Is this still something you'd want to have implemented @CanadaHonk ?

**Comment by CanadaHonk** [comment]
Yes, just isn't a high priority as I'm working on other things as well.

**Comment by Assassinduck** [comment]
I don't mind looking at it if you'd like, I would just need some quick clarification on some things in the codebase.

**Comment by CanadaHonk** [comment]
If you want. You can talk to me on Discord.
